.cdsrc.json

```diff
@@ -3,7 +3,7 @@
         "target": ".",
         "tasks": [
             {
-                "for": "java-cf"
+                "for": "java"
```

Comment:
are you sure about this? We still seem to run classic mtx in the bookshop

Reply:
I think this change is independent of streamlined or classic MTX, but we can also keep it back if you like.

Reply:
I'd suggest to go for it. @mofterdinger also proposed a similar change to the projects that we generate in the cds-services archetype.

Reply:
indeed` java-cf` and `java `point to the same implementation. `java-cf` should be replaced.